### PR TITLE
Update Italian Translation it_IT

### PR DESCRIPTION
**Pull request by StarFang208**
Here the new version of the italian localizaiton file.

### CODE REVIEW

**Comment by RedDevilus** [comment]
Perhaps change Italian localization update    (commit log)
to
Locales: Update Italian Translations it_IT

Otherwise seems to be good.

**Comment by StarFang208** [comment]
I changed the title now, as you asked.
Now it should be clearer.

**Comment by RedDevilus** [comment]
I didnt mean the title but the commit itself. 

image

Isnt a big deal though. Most important are the files itself.

File: locales/it_IT/pcsx2_Main.po [hunk 1]
```diff
@@ -2942,24 +2933,24 @@ msgid "Clear ISO list"
...
 #, c-format
 msgid "Slot %d - Unknown Time"
-msgstr ""
+msgstr "Slot %d - Tempo Sconosciuto"
 
 #: pcsx2/gui/Saveslots.h:113
 #, c-format
 msgid "Write time is %s %s."
-msgstr ""
+msgstr "Tempo scritto è %s %s."
```
Comment (mirh) [inline locales/it_IT/pcsx2_Main.po]: Tempo scrittura?
(mi sfugge cosa starebbe sta cosa)

**Comment by lightningterror** [comment]
@StarFang208 how's progress?

File: locales/it_IT/pcsx2_Iconized.po [hunk 2]
```diff
@@ -284,6 +284,24 @@ msgid ""
...
+"1) Più sicuro - Nessun hack di velocità. Impostazione più affidabile, ma "
+"possibilmente lenta.\n"
+"2) Sicuro - Predefinito. Alcuni hack di velocità noti per fornire "
+"potenziamenti, con minimo o nessun effetto collaterale.\n"
+"3) Bilanciato - Può aiutare quad core CPU.\n"
+"4) Aggressivo - Può aiutare CPU sottopotenziate su giochi meno impegnativi, "
+"ma rischia di causare problemi in altri casi.\n"
+"5) Molto aggressivo - Può aiutare cpu sottopotenziate su giochi meno "
+"impegnativi, ma è probabile che causi problemi in altri casi.\n"
+"6) Per lo più dannoso - applicazione dura di hack di velocità. Può aiutare "
```
[the remark this answers is not in the document]
Comment (StarFang208) [reply locales/it_IT/pcsx2_Iconized.po]: Lo modificherò in dannoso, specie se viene attivato su dispositivi deboli o vecchi.
Cosi come la dicitura hack di velocità che diventerà speed hack.

File: locales/it_IT/pcsx2_Iconized.po [hunk 3]
```diff
@@ -525,7 +550,7 @@ msgstr ""
 #: pcsx2/gui/Panels/MemoryCardListPanel.cpp:760
 #, c-format
 msgid "Failed: Destination memory card '%s' is in use."
-msgstr "Fallita! La memory card di destinazione '%s' è in uso"
+msgstr "Fallita! La memory card di destinazione '%s' è in uso."
```
[the remark this answers is not in the document]
Comment (StarFang208) [reply locales/it_IT/pcsx2_Iconized.po]: Credo che Poedit abbia scelto di testa propria di mettere quel punto esclamativo.

File: locales/it_IT/pcsx2_Iconized.po [hunk 4]
```diff
@@ -612,37 +637,49 @@ msgstr ""
 "per chiudere il pannello di configurazione."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:29
-#, c-format
+#, fuzzy, c-format
 msgid ""
 "50% cyclerate. Significant reduction of CPU requirements. Speedup for very "
 "lightweight games, slows down others. FMVs and audio may stutter or skip."
 msgstr ""
+"50% frequenza di ciclo. Riduzione significativa dei requisiti della CPU. "
```
[the remark this answers is not in the document]
Comment (StarFang208) [reply locales/it_IT/pcsx2_Iconized.po]: Ho rimpiazzato la linea con "cicli di macchina", accurato al contesto del programma.

File: locales/it_IT/pcsx2_Main.po [hunk 5]
```diff
@@ -926,34 +927,32 @@ msgstr "Informazioni su %s"
...
-#, fuzzy
 msgid "PlayStation 2 Emulator:"
-msgstr "Emulatore per PlayStation 2"
+msgstr "Emulatore per PlayStation 2:"
 
 #: pcsx2/gui/Dialogs/AboutBoxDialog.cpp:64
 msgid ""
 "Big thanks to everyone who contributed to the project throughout the years."
-msgstr ""
+msgstr "Ringrazio chiunque abbia contribuito al progetto nel corso degli anni."
```
[the remark this answers is not in the document]
Comment (StarFang208) [reply locales/it_IT/pcsx2_Main.po]: Modificato in "Ringrazio di cuore".

File: locales/it_IT/pcsx2_Iconized.po [hunk 6]
```diff
@@ -11,7 +11,7 @@ msgstr ""
 "Language: it_IT\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
-"Content-Transfer-Encoding: 8bit\n"
+"Content-Transfer-E ncoding: 8bit\n"
```
Comment (RedDevilus) [inline locales/it_IT/pcsx2_Iconized.po]: You are breaking this line

File: locales/it_IT/pcsx2_Main.po [hunk 7]
```diff
@@ -2463,17 +2464,17 @@ msgstr "Crea una memory card assegnata alla Porta selezionata."
 msgid "Delete memory file?"
 msgstr "Eliminare il file della memory card?"
 
+#: pcsx2/gui/Panels/MemoryCardListPanel.cpp:710
+msgid "Failed: Can only duplicate an existing card."
+msgstr "Fallita: Si possono copiare solo memory card esistenti."
```
[the remark this answers is not in the document]
Comment (StarFang208) [reply locales/it_IT/pcsx2_Main.po]: Sembra il menu di copia delle memory card, funzione simile a quella applicata ai singoli file dei giochi.

File: locales/it_IT/pcsx2_Main.po (hunk 1 above)
Comment (StarFang208) [reply locales/it_IT/pcsx2_Main.po]: Sembra che esiste per contare l'ora esatta in cui è stato scritto un file di salvataggio.

Tempo di scrittura è migliore rispetto a come lo avevo tradotto.

File: locales/it_IT/pcsx2_Iconized.po (hunk 6 above)
Comment (StarFang208) [reply locales/it_IT/pcsx2_Iconized.po]: Sorry, I fixed that in the new commit.

**Comment by RedDevilus** [comment]
Try to squash your files into 1 commit (guide in help channel in discord)

image

**Comment by StarFang208** [comment]
> 
> 
> @StarFang208 how's progress?

I fixed the translations according to this conversations' inputs.

**Comment by lightningterror** [comment]
> Try to squash your files into 1 commit (guide in help channel in discord)

It's fine, I can squash during merge.

**Comment by StarFang208** [comment]
Sorry, I forgot to do the squashing.

**Comment by RedDevilus** [comment]
It's fine then. Most important are quality translation and it can be handled , reviewed and then merged. Although the builds are failing somehow.

**Comment by lightningterror** [comment]
See https://travis-ci.org/github/PCSX2/pcsx2/jobs/665244342?utm_medium=notification&utm_source=github_status for other remaining issues.

**Comment by StarFang208** [comment]
The grammar mistakes and the lines' abrupt ends are fixed.

The build should be done without issues, now.

**Comment by lightningterror** [comment]
Mo files should be updated again with the new changes.

**Comment by RedDevilus** [comment]
Po files are for future translators.
Mo files are the translated languages to be used in PCSX2. So yeah it would break without re-uploading new ones.

**Comment by StarFang208** [comment]
Alright, here the new Mo files.

I hope they don't have the same issues with the Po files, as I don't have a program to read the code directly.

**Comment by RedDevilus** [comment]
Should work if u have the right poedit settings. (Wrong ones give bad line endings and mess with english strings) which i dont see in your files.

**Comment by StarFang208** [comment]
Good to know I followed the instructions properly.

**Comment by lightningterror** [comment]
@mirh you fine with the changes?

**Comment by mirh** [comment]
There is an odd double spacing ("  ") in cyclerate tips, but aside of that LGTM.

**Comment by bigol83** [comment]
The word Capture in main menu emulatorhas been mispelled as Capttura in Italian. 

It should be Cattura.

**Comment by StarFang208** [comment]
I fixed it for both the .po and .mo files.

**Pull request merged**